docs/readme.md

```diff
@@ -0,0 +1,109 @@
+Useful documentation, examples, and [recipes](./recipes/) to get you started.
```

Comment:
kinda think this should be `overview.md` or `basics.md`.

This kinda forces the user to choose a markdown file when entering /docs but i think that's good. it also means we don't have to maintain this file as a table-of-contents of all our docs